### DISCUSSION TITLE
722

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#906] [Comments]
# 722

최근에 영한님을 만나서 2~3시간 이야기를 나누었다. 영한님 차 타면서 집으로 가는 길에 예전에 함께 일하던 때에 대한 것들을 다루다보니 과거 생각이 많이 났다. 그러면서 그때 참 같이 일해서 좋았고, 선물 같은 시간이였다는 이야기를 나누었다. 다른 곳에서도 몇번 언급했지만, 영한님과의 인연은 페이스북 DM으로부터 시작이였다. 그 전까지는 컨퍼런스, 책으로만 뵙던 분을 DM으로 연락을 받았던 것이다. 그것도 내 블로그를 보시고 나서 말이다. 당시에 잘 다니고 있던 회사가 있었지만, 영한님을 비롯해서 당시에 유명하신 많은 분들과 함께 일할 수 있는 경험은 다시 없을것 같단 생각에 합류를 하게 됐다. 그리고 스타트업에서 성장하는 경험, 엔지니어링이 조직에 기여하는 법, 회사와 조직원이 동반 성장하는 경험 등..

https://jojoldu.tistory.com/722?fbclid=IwAR0X5SdB1cWZvZgTYQZLBbqNNZ1F_PxD6qbLcy9dmxBeQ2a7Y3k2t_rMOmE

**Comment by holyeye** [comment]
작은 인연의 조각들이 지금의 우리들의 선택에 참 많은 영향을 주었네요. 저도 삶에서 어느 시점에 뒤를 돌아보면 결국 동욱님 같은 좋은 분들을 만난 덕분에 더 나은 선택들을 할 수 있었던 것 같아요. 항상 응원합니다.

**Reply by jojoldu** [reply]
네 영한님 ㅋㅋㅋ 저도 너무 좋은 시간이였습니다 :) 
이제는 다른 형태로 같이 협업하게 되어서 또 새롭네요! ㅋㅋ
저도 항상 응원합니다 :)
